cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -368,6 +368,10 @@ public String parseLogging(Logging logging, ConfigMap externalCm) {
         }
     }
 
+    protected String addMonitorIntervalToExternalLogging(String data) {
+        return data;
```

Comment:
This looks a bit weird ... by the name it is supposed to add something, but it just returns what it gets. Is this something what still needs to be added?

Reply:
Yeah, it is my poor design. For all models, we do not need to add anything. This method is overridden in EO models to add `monitorInterval`.